rest/routes/user.py

```diff
@@ -117,18 +122,28 @@ def admin_editor_requests():
         ]
     :return: success status and json editor requests (if GET)
     """
-    if 'email' not in request.cookies:
-        return Response(response=json.dumps({}), status=401, mimetype='application/json')
+    if request.method == 'OPTIONS':
+        response = Response(response=json.dumps({}), status=200, mimetype='application/json')
+        response.headers.add('Access-Control-Allow-Origin', '*')
+        response.headers.add('Access-Control-Allow-Headers', 'email, content-type')
+        return response
+
+    if 'email' not in request.headers:
+        response = Response(response=json.dumps({}), status=401, mimetype='application/json')
+        response.headers.add('Access-Control-Allow-Origin', '*')
+        return response
 
-    email = request.cookies.get('email')
+    email = request.headers['email']
```

Comment:
if I understand it correctly I can set in my request header admin's email and got authenticated as an admin. You shouldn't base entire authentication on emails addresses because they are easy to guess. You should use session id or something like that. btw. I think it will be easier for you to implement authentication using JWT. What's more it will be easier to test it later in swagger, and easier to handle for our frontend team.

Reply:
I'll change it to JWT